### PR TITLE
Enable netfilter queue support

### PR DESCRIPTION
**Pull request by HiHat**
To support https://github.com/bol-van/zapret and https://github.com/Waujito/youtubeUnblock

### CODE REVIEW

**Comment by RMerl** [comment]
Asuswrt-Merlin devices based on kernel 4.1/4.19 have this built-in instead of as a module.  So for consistency I recommend that 2.6 also enables it built-in.

**Comment by HiHat** [comment]
Thanks!
Fixed!

File: release/src-rt-6.x.4708/linux/linux-2.6.36/config_base.6a [hunk 1]
```diff
@@ -557,7 +557,7 @@ CONFIG_NETFILTER_XT_MATCH_MAC=y
 CONFIG_NETFILTER_XT_MATCH_MARK=y
 CONFIG_NETFILTER_XT_MATCH_MULTIPORT=y
 # CONFIG_NETFILTER_XT_MATCH_OSF is not set
-# CONFIG_NETFILTER_XT_MATCH_OWNER is not set
```
Comment (SHAUKYY) [inline release/src-rt-6.x.4708/linux/linux-2.6.36/config_base.6a:560]: Is not set

File: release/src-rt-6.x.4708/linux/linux-2.6.36/config_base.6a [hunk 2]
```diff
@@ -482,7 +482,7 @@ CONFIG_NETFILTER_ADVANCED=y
 # Core Netfilter Configuration
 #
 CONFIG_NETFILTER_NETLINK=y
-# CONFIG_NETFILTER_NETLINK_QUEUE is not set
```
Comment (SHAUKYY) [inline release/src-rt-6.x.4708/linux/linux-2.6.36/config_base.6a:485]: Is not set

File: release/src-rt-6.x.4708/linux/linux-2.6.36/config_base.6a (hunk 1 above)
Comment (HiHat) [reply release/src-rt-6.x.4708/linux/linux-2.6.36/config_base.6a:560]: Yes, it was not set and required for specific iptables rules

File: release/src-rt-6.x.4708/linux/linux-2.6.36/config_base.6a (hunk 2 above)
Comment (HiHat) [reply release/src-rt-6.x.4708/linux/linux-2.6.36/config_base.6a:485]: Yes, it was not set and required for packet manipulation from user space processes.

**Pull request merged**